docs/alma/alma.rst

```diff
@@ -294,6 +294,55 @@ their types.
       velocity_resolution  double          m/s        Estimated velocity resolution from all the spectral windows, from frequency resolution.
 
 
+Query Results
+=============
+
+Results of queries are returned in tabular format. For convenience,
+the ''to_enhanced_table'' function can be used to have the initial result
+in a more useful format, i.e turn values into quantities, footprint into
+shape, etc. (Note: this require the ''shapes'' Python package to be installed.
```

Comment:
```suggestion
the ``to_enhanced_table`` function can be used to have the initial result
in a more useful format, i.e., turn values into quantities, footprint into
shape, etc. (Note: this require the `regions` Python package to be installed.
```
I might have the double vs single backticks backward... I always forget those... but the package should be regions, not shapes.